client/src/extended_bootstrap_css/bootstrap-wet-fixes-extensions.scss

```diff
@@ -268,25 +268,32 @@ div.label {
   background-color: $primaryColor !important;
   color: $textLightColor !important;
 }
-
-/* to be used with light-background */
-a.btn-ib-light {
-  color: inherit;
+.btn-ib-opened {
+  background-color: $backgroundColor;
+  border: 2px solid $backgroundColor;
+  color: $primaryColor !important;
+  border-radius: $standardBorderRadius;
+  margin: 0px 5px 0px 0px;
 }
+
+/* to be used with dark-background */
 .btn-ib-light {
-  background-color: inherit;
-  border: 2px solid $buttonPrimaryColor;
-  color: inherit;
-  border-radius: $standardBorderRadius; 
+  background-color: $primaryColor;
```

Comment:
This had a use, `background-color: inherit;` was so that we could use this class on any dark background, not just `$backgroundColor`, and have it look like it fit in. Was it necessary to change?

Reply:
It's not a necessary change, it's more like a clean up as there's no actual button that use this class, it's only used in anchor tag in igoc explorer which is not even used on dark background. I think it looks much better as `btn-ib-primary`, so I decided to basically repurpose it to use it like this

Reply:
I can revert it if you think it's necessary, but I don't see the point in leaving this class

Reply:
Hmm, ok. If it's for a particular use case, move it out of the common css and in to local css. Maybe give it a more specific name too.

Reply:
Well, I think the same style can potentially be used elsewhere in the future for more buttons with dark background? Even now, it's used in DisplayTable and rpb

Reply:
Doesn't `background-color: inherit;` === `background-color: $backgroundColor;`, in both cases? My point was that inheriting the parent's background colour would probably almost always mean having a background colour of `$backgroundColor`, but would also make it fit in seamlessly on other dark backgrounds if we happened to have any.

Reply:
Unless we had some jumbled HTML somewhere so that the inherited background colour wasn't the colour of the element it was positioned over, which was why I asked if this was a necessary change.

Reply:
ohhh yes I misinterpreted, it is both `$backgroundColor`, however, I needed it to be more explicit because of `btn-ib-light--reversed` so that the color is reversed explicitly

Reply:
So my point stands that it can stay in common css since this can still be used more in other places IMO, let me know if you think otherwise